english/wafflefries/wafflefries.md

```diff
@@ -0,0 +1,6 @@
+You sit down at the local bar and find some fries, the fries are given. 
+with a marshmellow sauce!?!?!
+
+[Dip the fry in the marshmellow?](../sleep/marshmallow.md)
+[Watch TV at the bar](www.youtube.com)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.